pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -182,6 +183,8 @@ type (
 		ID string `json:"id,omitempty"`
 		// +optional
 		CIDR *ipnet.IPNet `json:"cidr,omitempty"`
+		// +optional
```

Comment:
The generated schema will list the supported values if you add this comment:

```suggestion
		// Valid variants are `IPFamily` constants
		// +optional
```

I think you might also need to add a comment for the constants.